src/biophysics.jl

```diff
@@ -324,7 +367,19 @@ Q_ir_to_sub = A_tot * F_sub * ϵ_org * σ  * T_skin ^ 4
 (Q_ir_to_sky + Q_ir_to_sub)
 end
 
-function evap(
+radout(Body, p::Model, o::OrganismalVars, e::EnvironmentalVars) = begin
+    model_pars = stripparams(p)
+    org_pars = model_pars[1]
+    radout(o.T_surf, Body.geometry.area, org_pars.F_sky, org_pars.F_sub, org_pars.ϵ_org)
+end
+
+radout(T_x, Body, p::Model, o::OrganismalVars, e::EnvironmentalVars) = begin
+    model_pars = stripparams(p)
```

Comment:
Again we need to be dispatching on specific model kinds and / or body types in these methods.